ui/tests/acceptance/clients/counts/overview-test.js

```diff
@@ -314,9 +302,29 @@ module('Acceptance | clients | overview | sync not in license', function (hooks)
     assert.dom(GENERAL.tab('sync')).doesNotExist();
   });
 
-  test('it should hide secrets sync charts', async function (assert) {
-    assert.dom(CHARTS.chart('Secrets sync usage')).doesNotExist();
+  test('it should hide secrets sync stats', async function (assert) {
+    assert.dom(CLIENT_COUNT.statTextValue('Secret sync')).doesNotExist();
```

Comment:
In case it comes up in another review - this stat was intentionally labelled `Secret sync` (singular) because it's referring to the client type, not the secrets sync feature.